docs/assets/2.1-2.2-fix-delete-orphaned-resources.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -e
+
+echo "
+Deleting application broker"
+
+echo "Framefrogs please contribute"
+exit 1
+
+echo "
+Deleting rafter"
+
+kubectl -n kyma-system delete crd assetgroups.rafter.kyma-project.io
```

Comment:
do we want to make the script idempotent? As implemented with `set -e`, if it fails later for any reason, the fact it deleted the first CRD, every execution will fail as well because attempting to delete nonexisting resource with kubectl returns a non-zero exit code.

```bash
$ k get crd abcdefg
Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "abcdefg" not found

$ echo $?
1
```

Reply:
thx. I applied the fix in the new [PR](https://github.com/kyma-project/kyma/pull/14095).
I created a new one because this branch name is wrong (its going to be rollout in 2.3)